### ISSUE TITLE
Frustum culling call when it not needed

### ISSUE THREAD

**Issue by d954mas** [defold/defold#8088]
**Describe the bug (REQUIRED)**
I try to optimize my game. I look at profile and find that frustum culling for labels calls: 264 

Also i find that frustum culling for mesh and model calls(11) equals number of render.draw with frustum matrix

![2023-10-04_21-53-58](https://github.com/defold/defold/assets/2655263/57fb6301-e110-4611-9660-c505faa04bec)

https://github.com/defold/defold/blob/acd26d2bf4f4e37d14b9c28837b51e0a03ce8725/engine/render/src/render/render.cpp#L695

So it looks like you update frustum for all objects in render list, before check predicates. So when i render models you check frustum for all objects(models,labels, ...)
```lua
    render.draw(self.model_pred,frusutm_draw_opts)
```

**To Reproduce (REQUIRED)**
[TestFrustum.zip](https://github.com/defold/defold/files/12807582/TestFrustum.zip)
![2023-10-04_22-30-24](https://github.com/defold/defold/assets/2655263/a2040d14-0dfd-4167-aa46-166c014bdbf4)
```lua
  render.draw(self.model_pred,frusutm_draw_opts)
  render.draw(self.model2_pred,frusutm_draw_opts)
  render.draw(self.tile_pred,frusutm_draw_opts)
```
So 3 render.draw with frustum.
3 for Model and Sprite
6 for Label(2 gui)

**Expected behavior (REQUIRED)**
1. Do not update label frustum when i draw model.
2. Update frustum only for objects that i draw(by predicate)

**Defold version (REQUIRED):**
  - Version 1.6.0

**Platforms (REQUIRED):**
ALL

**Comment by JCash** [comment]
Yeah, I guess my attempt to avoid doing any superfluous frustum checks doesn't work in some scenarios.
I'll have a think around how to best approach this.

**Comment by AGulev** [comment]
maybe related https://github.com/defold/defold/issues/3528
